setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
Could you please add support for pylint to avoid some common errors. @andrey-canon